brigade/templates/base.html

```diff
@@ -30,7 +30,7 @@
 
         <nav class="nav-global-secondary">
           <ul>
-            <li><a href="/brigade/organize">Organize a Brigade</a></li>
+            <li><a href="/brigade/organize">Resources</a></li>
```

Comment:
As I mentioned on our call a little earlier, let's use the ``url_for()`` flask template tag for internal links.

Reply:
Sounds good, I'll do that. Since it is a bit unrelated to this change, I'm going to merge this first and then immediately address that separately.